Stripe/PublicHeaders/STPPaymentIntentShippingDetailsAddressParams.h

```diff
@@ -0,0 +1,64 @@
+//
+//  STPPaymentIntentShippingDetailsAddressParams.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 4/27/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPFormEncodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+Shipping address for a PaymentIntent's shipping details.
+
+@see https://stripe.com/docs/api/payment_intents/confirm#confirm_payment_intent-shipping-address
+*/
+@interface STPPaymentIntentShippingDetailsAddressParams : NSObject <NSCopying, STPFormEncodable>
+
+/**
+ City/District/Suburb/Town/Village.
+*/
+@property (nonatomic, copy, nullable, readwrite) NSString *city;
+
+/**
+ Two-letter country code (ISO 3166-1 alpha-2).
+ */
+@property (nonatomic, copy, nullable, readwrite) NSString *country;
+
+/**
+ Address line 1 (Street address/PO Box/Company name).
+ */
+@property (nonatomic, copy, readwrite) NSString *line1;
+
+/**
+ Address line 2 (Apartment/Suite/Unit/Building).
+ */
+@property (nonatomic, copy, nullable, readwrite) NSString *line2;
+
+/**
+ ZIP or postal code.
+ */
+@property (nonatomic, copy, nullable, readwrite) NSString *postalCode;
+
+/**
+ State/County/Province/Region.
+ */
+@property (nonatomic, copy, nullable, readwrite) NSString *state;
+
+/**
+ Initialize an `STPPaymentIntentShippingDetailsAddressParams` instance with required properties.
+ */
+- (instancetype)initWithLine1:(NSString *)line1;
+
+/**
+ Use `initWithLine1:` instead.
+ */
+- (instancetype)init NS_UNAVAILABLE;
```

Comment:
needs `new` marked as unavailable too (I'm, pretty sure)

Reply:
I've undone this change - makes sense to me but let's address in a separate PR with a story for how it will play with `+ decodedObjectFromAPIResponse` (should the pattern be that we add a private `initWithAPIResponse`?)